doc/releases/release-notes-3.7.rst

```diff
@@ -348,6 +353,14 @@ Boards & SoC Support
     optimal LVGL performance.
   * On ST STM32 boards, stm32cubeprogrammer runner can now be used to program external
     flash using ``--extload`` option.
+  * Added HEX file support for Linkserver to all NXP boards
+  * Updated the Linkserver west runner to reflect changes to the CLI of LinkServer v1.5.xx
+  * Added LinkServer support to NXP ``mimxrt1010_evk``, ``mimxrt1160_evk``, ``frdm_rw612``, ``rd_rw612_bga``, ``frdm_mcxn947``
+
+* Made these changes for RISC-V boards:
+
+* Made these changes for native/POSIX boards:
```

Comment:
```suggestion
```
These lines were removed in 1c8e46bb23f74d3223de0483d991dcdc806f1db7
Seems there was some issue in the rebase

Reply:
change addressed. I did rebase Friday after fetching the latest, no conflict was found